tools/testValidJson.ps1

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+<#
+    .SYNOPSIS
+        Get all JSON files recursively and test if they are valid by trying to import them.
+
+    .EXAMPLE
+        & $psEditor.GetEditorContext().CurrentFile.Path -WorkingDir '.\'
+#>
+
+# Input and expected output
+[OutputType([System.Void])]
+Param(
+    [Parameter()]
+    [ValidateScript({[System.IO.Directory]::Exists($_)})]
+    [string] $WorkingDir = '.\'
+)
+
+# PowerShell preferences
+$ErrorActionPreference = 'Stop'
+
+# Try to import all JSON files in the repo
+$TestValidJson = [PSCustomObject[]](
+    [System.IO.Directory]::GetFiles($WorkingDir, '*.json', [System.IO.SearchOption]::AllDirectories).ForEach{
```

Comment:
Linux was not happy about the default being `.\` (backslash): '/home/runner/work/vscode-powershell/vscode-powershell/vscode-powershell/.\'."